doc/source/install/install.md

```diff
@@ -64,11 +64,9 @@ components.
 
 | Category | Software version |
 | -------- | ---------------- |
-| Operating System | Ubuntu 16. |
-| Programming Language | Python 2.7.12 |
-| Automation | > Ansible 2.4 |
+| Operating System | Ubuntu 16. and Ubuntu 18. |
```

Comment:
get rid of dots

Reply:
I'd say Ubuntu 16.04 and 18.04.  We don't support other releases of Ubuntu 16 and 18.